app/templates/components/events/import-event.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="ui stackable centered grid">
+  <div class="twelve wide column">
+    <h3>Import Event</h3>
+    <hr class="hr-line">
```

Comment:
Actually ... Instead of this header line break combo ... Use https://semantic-ui.com/elements/header.html